colormath/color_conversions.py

```diff
@@ -724,12 +725,13 @@ def HSV_to_RGB(cobj, target_rgb, *args, **kwargs):
     else:
         raise ValueError("Unable to convert HSL->RGB due to value error.")
 
+    # TODO: Investigate intent of following code block.
```

Comment:
I'd suggest an issue tracker with a link to the line, pinned to a certain git sha (to prevent code shifting from breaking your link).